src/CommandTrait.php

```
@@ -0,0 +1,340 @@
+   * Then the command exit code should be 3
+   * @endcode
+   */
+  #[Then('the command exit code should be :code')]
+  public function commandAssertExitCode(string $code): void {
+    $this->commandAssertHasRun();
+
+    $expected = (int) $code;
+    $exit_code = (int) $this->commandExitCode;
+
+    if ($exit_code !== $expected) {
+      throw new \Exception(sprintf('Expected the command to exit with code %d, but it exited with code %d.', $expected, $exit_code));
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '198', column '17'). (undefined)

(MissingImport)